### DISCUSSION TITLE
my codes are not being pushed in github repository

### DISCUSSION THREAD

**Discussion by mamta-vyas** [community/community#140577] [Programming Help] [answered]
### Select Topic Area

General

### Body

git push origin main
Enumerating objects: 102, done.
Counting objects: 100% (102/102), done.
Delta compression using up to 4 threads
Compressing objects: 100% (91/91), done.
error: RPC failed; HTTP 400 curl 22 The requested URL returned error: 400
send-pack: unexpected disconnect while reading sideband packet
Writing objects: 100% (102/102), 1.10 MiB | 1.44 MiB/s, done.
Total 102 (delta 15), reused 0 (delta 0), pack-reused 0
fatal: the remote end hung up unexpectedly
Everything up-to-date

**Comment by davevad93** [comment]
Hi @mamta-vyas ,

It could be a HTTP buffering issue, try running this command:

```bash
git config http.postBuffer 524288000
```

**Comment by EfremMic** [comment, marked as answer]
Hei, try one of the following approach:
1, How about increasing the buffer size?

`git config --global http.postBuffer 524288000
`
2, Also, check if the file/ binary file you are trying to push is oversized.
You can do that by :

`git lfs track "*.file_extension"  # Example: "*.png"`

3, You can also check your config for remote URL 

If none of the above works, then you can try SSH instead of HTTPS. I hope this helps you.

**Comment by ebndev** [comment]
Hi @davevad93 & @EfremMic, thanks for taking the time to answer!

@mamta-vyas if the responses above helped, please feel encouraged to mark it as the answer. If not, let us know if you have further questions or post the solution if you've resolved it on your own. It would be helpful for other users who might experience this in the future 😄
